Makefile

```diff
@@ -36,8 +36,8 @@ modtidy:
 # Target: gen-proto                                                            #
 ################################################################################
 PROTOC ?=protoc
-PROTOC_VERSION = 24.4
-PROTOBUF_SUITE_VERSION = 24.4
+PROTOC_VERSION = 25.4
```

Comment:
Can we keep this the same as dapr/dapr?

Reply:
@cicoyle  How about we update the version in dapr instead?

Reply:
https://github.com/dapr/dapr/pull/8161

Reply:
fine by me